stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -1,24 +1,33 @@
-import React, { useEffect, useState, useCallback } from "react";
+import React, { useEffect, useState, useCallback, useReducer } from "react";
 import Components from "stockflux-components";
 import { Link } from "react-router-dom";
 import "./SecuritiesTable.css";
 import * as service from "../services/SecuritiesService";
 import ValidationError from "../services/ValidationError";
-import Alert from "./Alert";
+import Alert, { AlertType } from "./Alert";
 import { TableState } from "../enums";
-import Button from "./Button";
+import ToolTip from "./ToolTip";
+import Button, { ButtonSize } from "./Button";
+import { FaPen, FaTrashAlt, FaEye, FaEyeSlash } from "react-icons/fa";
```

Comment:
We should stick to a single icon theme, otherwise we risk things looking noticeably different. We use Font Awesome in the other apps so we should stick to those. Font Awesome calls the cross you're looking for "times" and the tick/check is called "check".

Reply:
Yep changed this